### PR TITLE
[DOCS][MINOR] Added blogs button in homepage

### PR DESCRIPTION
**Pull request by sagarlakshmipathy**
## *Important Read*
Added blogs button to site home

## What is the purpose of the pull request

UI change

## Brief change log

Added Blog button on site home

## Verify this pull request

Minor, visual
image

### CODE REVIEW

File: website/static/index.html [hunk 1]
```diff
@@ -32,7 +32,8 @@
       <a href="#" class="brand w-nav-brand"><img src="images/nav-logo.png" loading="lazy" alt="" class="nav-logo"></a>
       <nav role="navigation" class="nav-menu w-nav-menu">
         <a href="#" class="nav-link w-nav-link">Home</a>
-        <a href="https://onetable.dev/docs/setup/" class="nav-link w-nav-link">Docs</a><img src="images/break.svg" loading="lazy" alt="" class="image-10">
+        <a href="https://onetable.dev/docs/setup/" class="nav-link w-nav-link">Docs</a>
+        <a href="https://onetable.dev/blog" class="nav-link w-nav-link">Blog</a><img src="images/break.svg" loading="lazy" alt="" class="image-10">
```
Comment (the-other-tim-brown) [inline website/static/index.html:36]: should it be blog or blogs?

Comment (sagarlakshmipathy) [reply website/static/index.html:36]: just wanted to keep it consistent with the github folder structure. we could call it `Blogs` on the site.

**Comment by sagarlakshmipathy** [comment]
image
image

**Comment by sagarlakshmipathy** [comment]
I'm still keeping it as `blog` everywhere in the codebase except for the UI stuff. lmk if we need to change that.

**Review by the-other-tim-brown — APPROVED** [review]

**Pull request merged**